CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 <!-- next version -->
 
+## 0.105.0
+
+### 💡 Enhancements 💡
+
+- `collector`: Create automatically an OpenShift dashboard to visualize OpenTelemetry Collector metrics (#2995)
+  To enable this feature, you need to specify the `--openshift-create-dashboard` argument to the operator
+- `bundle`: Create an specific bundle for OpenShift environments. (#3054)
+- `collector`: Enabling ipFamilies and ipFamilyPolicy to be configured via OpenTelemetryCollector (#2958)
+- `auto-instrumentation`: Add Prometheus exporter to Python auto-instrumentation (#3122)
+
+### Components
+
+* [OpenTelemetry Collector - v0.105.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.105.0)
+* [OpenTelemetry Contrib - v0.105.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.105.0)
+* [Java auto-instrumentation - v1.32.1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.32.1)
```

Comment:
This needs to be updated https://github.com/open-telemetry/opentelemetry-operator/pull/3176

Reply:
also in versions.txt

Reply:
Just changed.